pulumi/grapl/__main__.py

```diff
@@ -368,11 +369,21 @@ def main() -> None:
             port=5532,
         )
 
+        uid_allocator_db = LocalPostgresInstance(
+            name="uid-allocator-db",
+            port=5532,
+        )
```

Comment:
Okay, lot to unpack here.

These port numbers should be unique, but it's the same one as plugin-work-queue.

So - what's actually happening at runtime is, we're using the same 1 postgres instance for both of these different services.

Personally? For local development? I think that's completely fine and okay, and that we should have 1 postgres instance for everything in local (and N postgres instances for everything in prod).